src/components/playback/playbackmanager.js

```diff
@@ -2573,8 +2574,15 @@ export class PlaybackManager {
             }
 
             const apiClient = ServerConnections.getApiClient(item.ServerId);
-            const mediaSourceId = playOptions.mediaSourceId || item.Id;
-            const getMediaStreams = apiClient.getItem(apiClient.getCurrentUserId(), mediaSourceId)
+            let mediaSourceId;
+
+            const isLiveTv = [BaseItemKind.TvChannel, BaseItemKind.Channel, BaseItemKind.LiveTvChannel].includes(item.Type);
```

Comment:
`Channel` refers to a special type of plugin that provides content "channels" so that should be excluded

```suggestion
            const isLiveTv = [BaseItemKind.TvChannel, BaseItemKind.LiveTvChannel].includes(item.Type);
```